tests/prague/eip7692_eof_v1/eip7620_eof_create/test_subcontainer_validation.py

```diff
@@ -468,3 +470,77 @@ def test_container_both_kinds_different_sub(eof_test: EOFTestFiller):
             kind=ContainerKind.INITCODE,
         ),
     )
+
+
+def test_deep_container(eof_test: EOFTestFiller):
+    """Test a very deeply nested container"""
+    container = Container(
+        sections=[
+            Section.Code(
+                code=Op.PUSH0 + Op.PUSH0 + Op.PUSH0 + Op.PUSH0 + Op.EOFCREATE[0] + Op.STOP,
+            ),
+            Section.Container(
+                container=Container(
+                    sections=[
+                        Section.Code(
+                            code=Op.PUSH0 + Op.PUSH0 + Op.RETURNCONTRACT[0],
+                        ),
+                        stop_sub_container,
+                    ]
+                )
+            ),
+        ]
+    )
+    last_container = container
+    while len(container) < MAX_INITCODE_SIZE:
+        last_container = container
+        container = Container(
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0 + Op.PUSH0 + Op.PUSH0 + Op.PUSH0 + Op.EOFCREATE[0] + Op.STOP,
+                ),
+                Section.Container(
+                    container=Container(
+                        sections=[
+                            Section.Code(
+                                code=Op.PUSH0 + Op.PUSH0 + Op.RETURNCONTRACT[0],
+                            ),
+                            Section.Container(container=last_container),
+                        ]
+                    )
+                ),
+            ]
+        )
+
+    eof_test(data=last_container)
+
+
+def test_wide_container(eof_test: EOFTestFiller):
+    """Test a container with the maximum number of sub-containers"""
```

Comment:
when we're at it, would it make sense to also test 257, 0x8000 and 0xffff (invalid) here? (The 0x8000 comes from a case fuzzing found recently for code section count [tests added here](https://github.com/ethereum/execution-spec-tests/pull/705))

Reply:
257 is doable and easy.  But 32k and 64k subcontainers make quite large tests (~8 Meg fixture)

Reply:
ah right, that other test was header only, while this has contents. Good point. OK so must add those separately.